tests/bluetooth/controller/common/src/helper_pdu.c

```diff
@@ -402,6 +407,79 @@ void helper_node_encode_cte_rsp(struct node_rx_pdu *rx, void *param)
 	rx->hdr.rx_ftr.iq_report = (struct cte_conn_iq_report *)param;
 }
 
+void helper_pdu_encode_cis_req(struct pdu_data *pdu, void *param)
+{
+	struct pdu_data_llctrl_cis_req *p = param;
+
+	pdu->ll_id = PDU_DATA_LLID_CTRL;
+	pdu->len = offsetof(struct pdu_data_llctrl, cis_req) +
+		   sizeof(struct pdu_data_llctrl_cis_req);
+	pdu->llctrl.opcode = PDU_DATA_LLCTRL_TYPE_CIS_REQ;
+
+	pdu->llctrl.cis_req.cig_id		=  p->cig_id;
+	pdu->llctrl.cis_req.cis_id		=  p->cis_id;
+	pdu->llctrl.cis_req.c_phy		=  p->c_phy;
+	pdu->llctrl.cis_req.p_phy		=  p->p_phy;
+	pdu->llctrl.cis_req.c_max_pdu		=  p->c_max_pdu;
+	pdu->llctrl.cis_req.p_max_pdu		=  p->p_max_pdu;
+	pdu->llctrl.cis_req.nse			=  p->nse;
+	pdu->llctrl.cis_req.p_bn		=  p->p_bn;
+	pdu->llctrl.cis_req.c_bn		=  p->c_bn;
+	pdu->llctrl.cis_req.c_ft		=  p->c_ft;
+	pdu->llctrl.cis_req.p_ft		=  p->p_ft;
+	pdu->llctrl.cis_req.iso_interval	=  p->iso_interval;
+	pdu->llctrl.cis_req.conn_event_count	= p->conn_event_count;
+	memcpy(pdu->llctrl.cis_req.c_max_sdu_packed, p->c_max_sdu_packed,
+	       sizeof(p->c_max_sdu_packed));
+	memcpy(pdu->llctrl.cis_req.p_max_sdu, p->p_max_sdu, sizeof(p->p_max_sdu));
+	memcpy(pdu->llctrl.cis_req.c_sdu_interval, p->c_sdu_interval, sizeof(p->c_sdu_interval));
+	memcpy(pdu->llctrl.cis_req.p_sdu_interval, p->p_sdu_interval, sizeof(p->p_sdu_interval));
+	memcpy(pdu->llctrl.cis_req.sub_interval, p->sub_interval, sizeof(p->sub_interval));
+	memcpy(pdu->llctrl.cis_req.cis_offset_min, p->cis_offset_min, sizeof(p->cis_offset_min));
+	memcpy(pdu->llctrl.cis_req.cis_offset_max, p->cis_offset_max, sizeof(p->cis_offset_max));
```

Comment:
Please only use spaces to indent after a non-tab character.
Conventionally we (void) cast the return values of the stdlib functions (like `memcpy`) if the return value isn't used. 

Please also keep lines below 80 when possible. 

```suggestion
	pdu->llctrl.cis_req.cig_id              = p->cig_id;
	pdu->llctrl.cis_req.cis_id              = p->cis_id;
	pdu->llctrl.cis_req.c_phy               = p->c_phy;
	pdu->llctrl.cis_req.p_phy               = p->p_phy;
	pdu->llctrl.cis_req.c_max_pdu           = p->c_max_pdu;
	pdu->llctrl.cis_req.p_max_pdu           = p->p_max_pdu;
	pdu->llctrl.cis_req.nse                 = p->nse;
	pdu->llctrl.cis_req.p_bn                = p->p_bn;
	pdu->llctrl.cis_req.c_bn                = p->c_bn;
	pdu->llctrl.cis_req.c_ft                = p->c_ft;
	pdu->llctrl.cis_req.p_ft                = p->p_ft;
	pdu->llctrl.cis_req.iso_interval        = p->iso_interval;
	pdu->llctrl.cis_req.conn_event_count    = p->conn_event_count;
	(void)memcpy(pdu->llctrl.cis_req.c_max_sdu_packed, p->c_max_sdu_packed,
	       sizeof(p->c_max_sdu_packed));
	(void)memcpy(pdu->llctrl.cis_req.p_max_sdu, p->p_max_sdu, 
		     sizeof(p->p_max_sdu));
	(void)memcpy(pdu->llctrl.cis_req.c_sdu_interval, p->c_sdu_interval, 
		     sizeof(p->c_sdu_interval));
	(void)memcpy(pdu->llctrl.cis_req.p_sdu_interval, p->p_sdu_interval, 
		     sizeof(p->p_sdu_interval));
	(void)memcpy(pdu->llctrl.cis_req.sub_interval, p->sub_interval, 
		     sizeof(p->sub_interval));
	(void)memcpy(pdu->llctrl.cis_req.cis_offset_min, p->cis_offset_min, 
		     sizeof(p->cis_offset_min));
	(void)memcpy(pdu->llctrl.cis_req.cis_offset_max, p->cis_offset_max, 
		     sizeof(p->cis_offset_max));
```

Reply:
Prefer to keep line length of max 100 as per guide line
Also, skipping (void) to keep style

Reply:
The guide line is to prefer 80, but will allow up to 100 where needed for better readability

Reply:
Well, fair enough re. guide line, however here I much prefer a long line for readability